p3387.tex

```diff
@@ -176,7 +176,7 @@ \subsection{odr-using parameters in \tcode{post}}
   // some other code; not a coroutine
 }
 \end{codeblock}
-Note that the user \emph{must} remove \tcode{const} from the parameter declaration on the definition of \tcode{f}, because otherwise, any attempt to move from \tcode{w} or otherwise modify \tcode{f} in the body of \tcode{w} cannot work because it would either not compile (if no \tcode{const_cast} is used) or be undefined behaviour as per \href{https://eel.is/c++draft/dcl.type.cv#4.sentence-1}{[dcl.type.cv]/4}. This is fine; it is possible to declare \tcode{w} is non-\tcode{const} on the definition even if an earlier declaration of \tcode{f} declares it as \tcode{const}.
+Note that the user \emph{must} remove \tcode{const} from the parameter declaration on the definition of \tcode{f}, because otherwise, any attempt to move from \tcode{w} or otherwise modify \tcode{f} in the body of \tcode{f} cannot work because it would either not compile (if no \tcode{const_cast} is used) or be undefined behaviour as per \href{https://eel.is/c++draft/dcl.type.cv#4.sentence-1}{[dcl.type.cv]/4}. This is fine; it is possible to declare \tcode{w} is non-\tcode{const} on the definition even if an earlier declaration of \tcode{f} declares it as \tcode{const}.
```

Comment:
I think that should actually say "modify w in the body of f". I had it the wrong way around but it's still incorrect now

Reply:
fixed